include/vcpkg/build.h

```diff
@@ -204,17 +204,33 @@ namespace vcpkg
 
     struct BuildResultCounts
     {
-        int succeeded = 0;
-        int build_failed = 0;
-        int post_build_checks_failed = 0;
-        int file_conflicts = 0;
-        int cascaded_due_to_missing_dependencies = 0;
-        int excluded = 0;
-        int cache_missing = 0;
-        int downloaded = 0;
-        int removed = 0;
-
-        void increment(const BuildResult build_result);
+        unsigned int succeeded = 0;
+        unsigned int build_failed = 0;
+        unsigned int post_build_checks_failed = 0;
+        unsigned int file_conflicts = 0;
+        unsigned int cascaded_due_to_missing_dependencies = 0;
+        unsigned int excluded = 0;
+        unsigned int cache_missing = 0;
+        unsigned int downloaded = 0;
+        unsigned int removed = 0;
+
+        constexpr void increment(const BuildResult build_result) noexcept
```

Comment:
I disagree that this is an improvement. There is no reason to ever make a constexpr one of these and being able to have the definition out of line is good.